index-scheduler/src/index_mapper/mod.rs

```diff
@@ -360,6 +404,45 @@ impl IndexMapper {
         Ok(())
     }
 
+    /// The stats of an index.
+    ///
+    /// If available in the cache, they are directly returned.
+    /// Otherwise, the `Index` is opened to compute the stats on the fly (the result is not cached).
```

Comment:
Is this possible?
From what I see, we're always creating and updating the stats of the indexes; thus, we should probably return an internal error instead of computing the stats silently?

Reply:
It happens when e.g. using a 1.0 DB with a 1.1 Meilisearch.

Reply:
Yeah, I prefer that we continue to be resilient to missing stats. And I like that we only try to cache it if it is present (we could wait for too long anyway).

Reply:
> It happens when e.g. using a 1.0 DB with a 1.1 Meilisearch.

But that won't happen since you need to use a dump to move from 1.0 to 1.1.

However, if you both prefer keeping it this way, we can. I just feel like it could hide a bug.

Reply:
> But that won't happen since you need to use a dump to move from 1.0 to 1.1.

Will we actually need to use a dump?

If adding this DB is the only change we did to the data structures, I'd be OK to allow using a 1.0 DB from a 1.1 Meilisearch.